scipy/optimize/__init__.py

```diff
@@ -110,7 +110,8 @@
 .. autosummary::
    :toctree: generated/
 
-   fixed_point - Single-variable fixed-point solver
+   fixpoint - Single-variable fixed-point solver
+   fixed_point - Steffensen's method for fixed points
```

Comment:
names sound a bit confusing, difference looks to me more like a typo than anything that would give a hint for the actual difference.
(I don't have any alternative suggestion.)